src/org/oidc/service/base/enums/Algorithm.java

```diff
@@ -1,4 +1,4 @@
-package enums;
+package org.oidc.service.base.enums;
```

Comment:
Like I mentioned yesterday , lot of the enums would be shared across. For example, Algorithm enum. Let's keep this in org.oidc.common package.

Reply:
moved all enums to org.oidc.common